main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1471,9 +1471,10 @@ private void updateWatchlistBox() {
          */
         private void updateFavPointBox() {
             // Favorite counts
-            if (cache.getFavoritePoints() > 0) {
+            final int favCount = cache.getFavoritePoints();
+            if (favCount >= 0) {
                 favoriteLine.left.setVisibility(View.VISIBLE);
-                favoriteLine.right.setText(cache.getFavoritePoints() + "×");
+                favoriteLine.right.setText(favCount + "×");
```

Comment:
We should introduce (or reuse?) a resource string for the text.

Reply:
"x" will be removed from here in #7441

Reply:
We should use the same style to show favorite counts at all places. Removing the `×`leads to an inconsistent style.

Reply:
Actually I was thinking about it, but now it's different style. For CachePopup appearance is not changing, I find just a number there looking a bit ugly.
For CacheDetail though it will have percentage all the time except zero. And zero without x looks fine together will other numbers IMO. And of course when there is percentage, no need in x any more

Reply:
If you think that we can remove x from cachepopup as well - can do, just seemed not so nice from my point of view

Reply:
Don't get me wrong. I think, we should keep the `×`.
My point is that we should use the same style here and in `AbstractDialogFragment.java `.

Reply:
>For CacheDetail though it will have percentage all the time except zero. And zero without x looks fine together will other numbers IMO.

Reply:
Adding percentages is another PR – please focus on this one.